umm-spec-lib/src/cmr/umm_spec/migration/version/collection.clj

```diff
@@ -434,3 +436,22 @@
                              (into []
                                (remove #(= "GET CAPABILITIES" (:Type %)) related-urls))))
       util/remove-nils-empty-maps-seqs))
+
+;; Migrations related to 1.16.4 --------------
+
+(defmethod interface/migrate-umm-version [:collection "1.16.3" "1.16.4"]
+  [context collection & _]
+  ;; No need to migrate
+  collection)
+
+(defn- remove-1-16-4-urls
+  [related-urls]
+  (let [sans ["HITIDE", "SOTO", "Sub-Orbital Order Tool", "CERES Ordering Tool"]]
+    (into [] (remove #(not (= nil (some #{(:Subtype %)} sans))) related-urls))))
```

Comment:
This is a bit hard to read, can we simplify here?  We are doing not equal to nil?  
Perhaps this helps https://clojuredocs.org/clojure.core/some_q

Reply:
I would love to simplify that line, I'll take a look at some?.